genestack_client/cla.py

```diff
@@ -191,10 +191,31 @@ def create_merged_reads(self, sources_folder, grouping_key):
 
 # preprocess microarrays
 
+class AffymetrixMicroarraysNormalizationApplication(CLApplication):
+    APPLICATION_ID = 'genestack/affymetrix-normalization'
+
+
+class AgilentMicroarraysNormalizationApplication(CLApplication):
+    APPLICATION_ID = 'genestack/agilent-normalization'
+
+
+class L1000MicroarraysNormalizationApplication(CLApplication):
+    APPLICATION_ID = 'genestack/l1000-normalization'
+
+
+class GenePixMicroarraysNormalizationApplication(CLApplication):
+    APPLICATION_ID = 'genestack/genepix-normalization'
+
+
+class InfiniumMicroarraysNormalizationApplication(CLApplication):
+    APPLICATION_ID = 'genestack/infinium-methylation-normalization'
+
+
+# TODO: remove, deprecated. We perform renaming in a scope of global 's' -> 'z' refactoring (american style english).
```

Comment:
We need to use old class name (to fix usages) but with new file id:

Better way to do it is assign class to variable
```
TODO: Deprecated at {current} will be removed at {current} + 3 ....
AffymetrixMicroarraysNormalisationApplication = AffymetrixMicroarraysNormalizationApplication
....
```